src/test/java/hudson/plugins/git/GitSCMTest.java

```diff
@@ -1637,9 +1638,7 @@ public void testBlankRepositoryName() throws Exception {
     @Test
     public void testSubmoduleFixup() throws Exception {
         /* Unreliable on Windows and not a platform specific test */
-        if (isWindows()) {
-            return;
-        }
+        Assume.assumeFalse(isWindows());
```

Comment:
Thanks for the fix!

These tests intentionally use the `if` construct rather than the `Assume` construct because the `Assume` construct generates a warning about skipped tests that then is shown as a warning in the warnings ng plugin output.  I'd prefer to continue using the early exit without using a JUnit construct so that the warning is not generated.

Alternately, if the `waitForCompletion` resolves the flakiness in the test, then let's remove the early return and let the test run on Windows.

Reply:
I've started a test series on my infra to see if the `Assume` can be removed for Windows.  Will report the results here.